diff --git a/node_modules/@react-navigation/elements/lib/module/ResourceSavingView.js b/node_modules/@react-navigation/elements/lib/module/ResourceSavingView.js
index eb063ab..d696374 100644
--- a/node_modules/@react-navigation/elements/lib/module/ResourceSavingView.js
+++ b/node_modules/@react-navigation/elements/lib/module/ResourceSavingView.js
@@ -17,26 +17,24 @@ export function ResourceSavingView({
     , {
       hidden: !visible,
       style: [{
-        display: visible ? 'flex' : 'none'
+        display: visible ? 'flex' : 'none',
+        pointerEvents: visible ? 'auto' : 'none'
       }, styles.container, style],
-      pointerEvents: visible ? 'auto' : 'none',
       ...rest,
       children: children
     });
   }
   return /*#__PURE__*/_jsx(View, {
-    style: [styles.container, style]
+    style: [styles.container, { pointerEvents: visible ? 'auto' : 'none' }, style]
     // box-none doesn't seem to work properly on Android
     ,
-    pointerEvents: visible ? 'auto' : 'none',
     children: /*#__PURE__*/_jsx(View, {
       collapsable: false,
       removeClippedSubviews:
       // On iOS & macOS, set removeClippedSubviews to true only when not focused
       // This is an workaround for a bug where the clipped view never re-appears
       Platform.OS === 'ios' || Platform.OS === 'macos' ? !visible : true,
-      pointerEvents: visible ? 'auto' : 'none',
-      style: visible ? styles.attached : styles.detached,
+      style: [visible ? styles.attached : styles.detached, { pointerEvents: visible ? 'auto' : 'none' }],
       children: children
     })
   });
diff --git a/node_modules/@react-navigation/elements/lib/module/Screen.js b/node_modules/@react-navigation/elements/lib/module/Screen.js
index 5a7086e..ccb4ae5 100644
--- a/node_modules/@react-navigation/elements/lib/module/Screen.js
+++ b/node_modules/@react-navigation/elements/lib/module/Screen.js
@@ -46,14 +46,13 @@ export function Screen(props) {
       navigation: navigation,
       children: /*#__PURE__*/_jsx(View, {
         ref: headerRef,
-        pointerEvents: "box-none",
         onLayout: e => {
           const {
             height
           } = e.nativeEvent.layout;
           setHeaderHeight(height);
         },
-        style: [styles.header, headerTransparent ? styles.absolute : null],
+        style: [styles.header, headerTransparent ? styles.absolute : null, { pointerEvents: 'box-none' }],
         children: header
       })
     }) : null, /*#__PURE__*/_jsx(View, {
